src/hb-aat-layout.cc

```diff
@@ -301,3 +309,29 @@ _hb_aat_language_get (hb_face_t *face,
 {
   return _get_ltag (face).get_language (i);
 }
+
+/**
+ * hb_aat_get_feature_settings:
+ * @face:            a font face.
+ * @identifier:      aat feature id you are querying.
+ * @default_setting: (out): default value for the type. If it is HB_AAT_FEATURE_NO_DEFAULT_INDEX
+ *                          means non is default and it is not exclusive also.
+ * @start_offset:    start offset, if you are iterating
```

Comment:
I don't fully understand what start_offset points at? What would the client iterate over?

Reply:
Somehow for API consistency and correctness, we need to consider if input is shorter than settings available so giving the client to ask from a different position with their limited buffer. See the test to see how it can be used. Given the low of number of options available here, 0 is what will clients will use most of the time but just like the similar other situation we need to do that consistently I believe.